dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/view/HostView.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.rest.api.v1.secret.view;
+
+public class HostView {
```

Comment:
Maybe we can rename this to `SiteView` as the term Host should not be used anymore, and use Site instead, just like we see it in the back-end UI.